roxie/ccd/ccdqueue.cpp

```diff
@@ -323,7 +318,7 @@ void openMulticastSocket()
             DBGLOG("Roxie: multicast socket buffer size could not be set (requested=%d actual %d", udpMulticastBufferSize, actualSize);
             throwUnexpected();
         }
-        if (traceLevel)
+        if (doTrace(TraceFlags::All))
```

Comment:
Not sure TraceFlags::All conveys the actual meaning.  
Can we either make the feature flag param optional as well, or relabel ALL to IGNORE or something similar?

Reply:
The meaning is "Always trace, regardless of the feature flags"... As for the name, I don't care, but ignore seems even more confusing!